package.json

```diff
@@ -84,7 +84,8 @@
     "react-qr-code": "^2.0.7",
     "react-redux": "^8.0.1",
     "styled-components": "^5.3.5",
-    "victory-native": "^36.6.10"
+    "victory-native": "^36.6.10",
+    "expo-splash-screen": "~0.20.5"
```

Comment:
This was automatically added I suppose. But we don't need it, right?

Reply:
Yep.. removed. I hope it doesn't come back 🌬️